src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
+  font-family: Roboto, Arial, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 0 50px;
+  max-width: 1200px;
+}
+
+.go-home {
+  margin: 10px 0;
+  height: 40px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+}
```

Comment:
> you don't need this prop, because you setting flex-box to `header`

Automated tests require two separate classes: .nav and .nav__list. I have to specify them and give them this feature and 100% height, because clickable area above and below the text.